frontend/app/routes/_gcweb-app.personal-information._index.tsx

```diff
@@ -67,11 +69,16 @@ export default function PersonalInformationIndex() {
           title={t('personal-information:index.home-address')}
           icon="glyphicon-map-marker"
         >
-          {homeAddressInfo ? (
-            <Address address={homeAddressInfo.address} city={homeAddressInfo.city} provinceState={homeAddressInfo.province} postalZipCode={homeAddressInfo.postalCode} country={homeAddressInfo.country} />
-          ) : (
-            <p>{t('personal-information:index.no-address-on-file')}</p>
+          {homeAddressInfo && (
+            <Address
+              address={homeAddressInfo?.address}
+              city={homeAddressInfo?.city}
+              provinceState={regionList.find((region) => region.code === homeAddressInfo.province)?.nameEn}
+              postalZipCode={homeAddressInfo?.postalCode}
+              country={countryList.find((country) => country.code === homeAddressInfo.country)?.nameEn ?? ' '}
```

Comment:
This will always render the English name for a province and country even if the user toggled this page to French.

This same comment can be applied throughout this PR where we only render the `nameEn` field of a country/region.

Reply:
fixed